docker-compose.yaml

```diff
@@ -5,15 +5,20 @@ services:
     container_name: herold_bot
     image: h0d0user/herold_bot:latest
     restart: always
-    network_mode: host
+    networks:
+      - herold_net
     environment:
       - API_ID=YOURAPIID
       - API_HASH=YOURAPIHASH
       - SESSION=YOURSESSION
     volumes:
-      - "/root/herold_bot/circulation_ids_list.txt:/opt/circulation_ids_list.txt"
-      - "/root/herold_bot/circulation_response.txt:/opt/circulation_response.txt"
-      - "/root/herold_bot/new_year_patterns.txt:/opt/new_year_patterns.txt"
+      - "/root/herold_bot/herold_database.db:/opt/herold_database.db"
+      - "/root/herold_bot/congrats_file.txt:/opt/congrats_file.txt"
       - "/root/herold_bot/JustTalk.session:/opt/JustTalk.session"
+      - "/root/herold_bot/settings.toml:/opt/settings.toml"
```

Comment:
The `setting.toml` is only config file. The `congrats_file.txt` store congrats strings. The `session` file required to work with telegram. And the last one is `db`. For now all of it will store in container in one place. Later it is possible to put all this in different locations